pkg/fleet/env/env.go

```diff
@@ -108,13 +128,51 @@ func (e *Env) ToEnv() []string {
 	if e.RegistryAuthOverride != "" {
 		env = append(env, envRegistryAuth+"="+e.RegistryAuthOverride)
 	}
+	if len(e.ApmLibraries) > 0 {
+		b := strings.Builder{}
+		b.WriteString(envApmLibraries)
+		b.WriteString("=")
+		for l, v := range e.ApmLibraries {
+			b.WriteString(string(l))
+			if v != "" {
+				b.WriteString(":")
+				b.WriteString(string(v))
+			}
+			b.WriteString(",")
+		}
+		envVar := b.String()
+		env = append(env, envVar[:len(envVar)-1])
+	}
 	env = append(env, overridesByNameToEnv(envRegistryURL, e.RegistryOverrideByImage)...)
 	env = append(env, overridesByNameToEnv(envRegistryAuth, e.RegistryAuthOverrideByImage)...)
 	env = append(env, overridesByNameToEnv(envDefaultPackageInstall, e.DefaultPackagesInstallOverride)...)
 	env = append(env, overridesByNameToEnv(envDefaultPackageVersion, e.DefaultPackagesVersionOverride)...)
 	return env
 }
 
+func parseApmLibrariesEnv() map[ApmLibLanguage]ApmLibVersion {
+	apmLibraries := os.Getenv(envApmLibraries)
+	if apmLibraries == "all" {
+		return map[ApmLibLanguage]ApmLibVersion{
```

Comment:
<!-- Thoughts represent an idea that popped up from reviewing. These comments are non-blocking by nature, but they are extremely valuable and can lead to more focused initiatives and mentoring opportunities.  -->
**💭 Thought:** ‏Can't we just return an empty map to leverage the defaults?

Reply:
I think the end goal is that this env variable will decide which libraries are installed when running the install script. 
So we want to be able to differentiate bewteen
* This language was present in the env list (or implicitely with `all`) with an empty version
* This library was not present and should not be installed

Reply:
What I mean is:
- If a package is not set in this map, its version is assumed to be `latest`
- The packages available are already laid out in `defaultPackages`

So we don't need to list them again here when the env var is set to `all`

Not really a big deal here, but that'd avoid mistakes where we update one place but forget the other when adding a new language